grails-app/controllers/org/pih/warehouse/apitablero/ApitableroController.groovy

```diff
@@ -13,39 +17,69 @@ class ApitableroController {
     def indicatorDataService
     def inventorySnapshotService
 
-    def getNumberData = {
+    def getInventoryByLotAndBin = {
```

Comment:
Oh nice, I think you already split the numberData endpoint into multiple endpoints. Sweet. Ignore my comment above.